### PR TITLE
Configurable sync interval

### PR DESCRIPTION
**Pull request by erikvatt**
Proposed changes:
- sync interval configurable with either `sync-interval` flag or `SYNC_INTERVAL_MINUTES` env
- add templating function for optionally base64 encoding values

### CODE REVIEW

File: internal/template/template.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"bytes"
 	"text/template"
 
+	b64 "encoding/base64"
```
Comment (jhrv) [inline internal/template/template.go:7]: maybe not necessary to alias this

File: main.go [hunk 2]
```diff
@@ -60,13 +61,15 @@ func main() {
...
 	var debug bool
+	var syncInterval string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&enableWebhooks, "enable-webhooks", true, "Enable webhooks")
 	flag.BoolVar(&debug, "debug", os.Getenv("DEBUG") == "true", "Enable debug logging")
+	flag.StringVar(&syncInterval, "sync-interval", os.Getenv("SYNC_INTERVAL_MINUTES"), "Syncronization interval for reconciliation in minutes")
```
Comment (jhrv) [inline main.go:72]:
```suggestion
	flag.StringVar(&syncInterval, "sync-interval", os.Getenv("SYNC_INTERVAL_MINUTES"), "Synchronization interval for reconciliation in minutes")
```

**Review by jhrv — COMMENTED** [review]
lgmt! maybe also add something in the README about the new templating features.

**Comment by erikvatt** [comment]
Created an issue for loading the secrets referenced in the replication config https://github.com/nais/replicator/issues/6

**Pull request merged**